src/constants.js

```diff
@@ -0,0 +1,6 @@
+const constants = Object.freeze({
+    pageOrderHistory: "order-history",
+    pageRestaurants: "restaurants"
+});
+
+export { constants };
```

Comment:
Premature use of contants file maybe?
(I am sensing an "anit-magic-strings" thing overall)

Reply:
In the latest branch both of these are used in a multiple files. Another option is a single module to make all the api calls. I feel like for the course keeping things somewhat close is helpful to seeing what happens, in a larger application I'd tend to abstract things more.

But you're correct I tend to avoid magic strings, numbers, etc.